Mathlib/Data/Set/Image.lean

```diff
@@ -607,6 +607,13 @@ theorem image_univ {f : α → β} : f '' univ = range f := by
   ext
   simp [image, range]
 
+lemma image_compl_eq_range_sdiff_image {f : α → β} (hf : Injective f) (s : Set α) :
```

Comment:
sdiff?

Reply:
All of mathlib calls it `sdiff`, except the `Set` API. Let's stop adding more technical debt